6_handbook/0_handbook_intro.txt

```diff
@@ -1,4 +0,0 @@
-== Handbook
-
-This chapter contains some tips and tricks you might want to know about using Rudder in a production environment, with some useful optimization and procedures.
-
```

Comment:
Why remove the intro? I'm not against it, I'd just like to understand the rationale.